io.github.hakuneko.HakuNeko.json

```diff
@@ -0,0 +1,54 @@
+{
+	"app-id": "io.github.hakuneko.HakuNeko",
+	"base": "io.atom.electron.BaseApp",
+	"base-version": "stable",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "19.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "hakuneko",
+	"separate-locales": false,
+	"rename-desktop-file": "hakuneko-desktop.desktop",
+	"rename-icon": "hakuneko-desktop",
+	"finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home"
+	],
+	"modules": [
+		{
+			"name": "hakuneko-desktop",
+			"buildsystem": "simple",
+			"build-commands": [
+				"ar x hakuneko-desktop_*.deb",
+				"rm -f hakuneko-desktop_*.deb",
+				"tar xf data.tar.xz",
+				"rm -f control.tar.gz data.tar.xz debian-binary",
+				"cp -r usr/* /app",
+				"chmod -R a-s,go+rX,go-w /app/lib/hakuneko-desktop/hakuneko",
+				"install hakuneko.sh /app/bin/hakuneko",
+				"install -Dm644 io.github.hakuneko.HakuNeko.metainfo.xml /app/share/appdata/io.github.hakuneko.HakuNeko.metainfo.xml"
+			],
+			"sources": [
+				{
+					"type": "file",
+					"url": "https://github.com/manga-download/hakuneko/releases/download/v6.1.7/hakuneko-desktop_6.1.7_linux_amd64.deb",
+					"sha256": "06bb17d7a06bb0601053eaaf423f9176f06ff3636cc43ffc024438e1962dcd02"
+				},
+				{
+					"type": "script",
+					"dest-filename": "hakuneko.sh",
+					"commands": [
+						"exec env TMPDIR=$XDG_CACHE_HOME /app/lib/hakuneko-desktop/hakuneko \"$@\""
+						    ]
```

Comment:
Nitpick: indentation is wrong here

Reply:
I think I finally got the Indention right after some tries.